test/core/core_test.cpp

```diff
@@ -0,0 +1,81 @@
+#include <gtest/gtest.h>
+#include "core/core.cpp"
+
+    const std::string jsonInput = R"(
```

Comment:
format it
also, this could be inside anonymous namespace, test suite (if more than one test case) or inside test case body (if only one test case uses the data)